README.md

```diff
@@ -113,8 +113,6 @@ to see all available versions.
 | Name             | Mail Address                | GitHub Profile                                |
 |------------------|-----------------------------|-----------------------------------------------|
 | Nicholas Strydom | nstrydom@gmail.com          | [nstrydom2](https://github.com/nstrydom2)     |
-| hentai-chan      | dev.hentai-chan@outlook.com | [hentai-chan](https://github.com/hentai-chan) |
```

Comment:
Keep this one at the very least. This collaborator has provided a lot of useful and robust contributions.